src/server/services/tests/BruincastServices.test.js

```diff
@@ -4,12 +4,114 @@ require('babel-polyfill');
 const BruincastServices = require('../BruincastServices');
 const client = require('../../models/db');
 
-const testCollectionName = 'crossliststests';
+const testCrosslistServicesCollectionName = 'crossliststestservices';
+const testAnalyticsCrosslistsCollectionName = 'crossliststestanalytics';
+const testAnalyticsCastsCollectionName = 'bruincastmediatestanalytics';
+const testAnalyticsPlaybacksCollectionName = 'playbackstestanalytics';
 
 beforeAll(async done => {
   const dbURL = `${process.env.DB_URL}${process.env.DB_DATABASE}?replicaSet=${process.env.DB_REPLSET}`;
   await client.connect(dbURL);
-  await client.db(process.env.DB_DATABASE).createCollection(testCollectionName);
+  const db = client.db(process.env.DB_DATABASE);
+  await db.createCollection(testCrosslistServicesCollectionName);
```

Comment:
@maxpluez What if there's an error in the test and the cleanup afterAll() doesn't get called? Can you add in a check here that will make sure the collection is empty as well?

https://docs.mongodb.com/manual/reference/method/db.collection.remove/#remove-all-documents-from-a-collection

Also, you should be able to put this in a loop so you don't have to repeat the code to call createCollection and remove. You can then use the same loop in afterAll() as well to delete the collections.

Reply:
I just tested that afterAll is called even if it fails. The collections are removed if tests failed.

What do you mean by 'put this in a loop'? Do you mean creating an array of collection names to loop through?

Reply:
@maxpluez  yeah in a loop. But if this works even if a test fails then it’s okay